src/Base/StlFeatures.h

```diff
@@ -72,6 +73,10 @@ make_unique(std::size_t n)
 template<class T, class... Args>
 inline typename Detail::UniqueIf<T>::KnownBound
 make_unique(Args&&...) = delete;
+#else
+// Visual Studio 2015 implements make_unique
```

Comment:
IIIRC, MSVS2010 was able to support of some C++11 features (but not all, of course).
Also there was MSVS2012.
But make_unique is a C++14 feature, and it supported in MSVS2013 and MSVS2015 only.
It will be better to check the value of _MSC_VER macro for compiler version.